### ISSUE TITLE
[BUG] >=2.0 no longer compiles against pypy3

### ISSUE THREAD

**Issue by tgbugs** [Marco-Sulla/python-frozendict#45]
OS version:
Gentoo Linux (rolling)

p version (python3 -V -V):
Python 3.8.12 (9ef55f6fc3697bf8d85334ce2d70fdbba888e3ab, Jan 10 2022, 23:10:59)
[PyPy 7.3.7 with GCC 10.3.0]

Steps to reproduce:

1. pypy3 setup.py build
2. compile fails because certain cpython includes are used, though commenting them out doesn't seem to impact a normal python3_8 build, this does not happen with any of the 1.x release.

Actual result:

```
python-frozendict/frozendict/src/3_8/cpython_src/Include/internal/pycore_pystate.h:11:10: fatal error: cpython/initconfig.h: No such file or directory
   11 | #include "cpython/initconfig.h"
      |          ^~~~~~~~~~~~~~~~~~~~~~
```

The 3_8 c code seems to be quite divergent from the code for 3_9 and others, so I'm not entirely sure what is going on here.

**Comment by Marco-Sulla** [comment]
Thank you for the report. 

##TL;DR

This will be fixed in the next release

##Long explaining
I suppose that is the C Extension that does not build on Pypy. Indeed I turned on the compilation of the C Extension by default in `frozendict` 2.0. 

Pure py builds can be retrieved from Pypi. I supposed you can also create it from code running `pypy3 setup.py py bdist_wheel` (don't know for sure since I don't have Pypy).

Anyway I want to make the compilation of the C Extension optional in the next release, so the problem will be fixed.

**Comment by Marco-Sulla** [comment]
I'm just re-thinking about this decision. Do you think it's too bad if I leave the things as they are, and I add in the README a doc about the `py` parameter for `setup.py`?

The problem is I would get feedback for the C Extension. I'm releasing binaries for Linux, Windows and MacOS, CPython 3.6+. If I make it optional, it's more difficult to get some info if it doesn't work on some machine.

Furthermore I want to investigate why it does not build on Pypy.

**Comment by tgbugs** [comment]
I think it is probably the correct decision to build the extension by default and provide/document an option to disable building it. That would let me work around the issue on pypy for now while making progress toward having it work on all implementations.

**Comment by Marco-Sulla** [comment]
Thank you for your willingness. New version released :)
